### PR TITLE
New version svtplay-dl 0.30.2016.01.10

### PR DESCRIPTION
**Pull request by spaam**
i added so the script depends on python.
i guess i need to do that if i want to use python from homebrew in the shebang instead of /usr/bin/python?

right now i have "#!/usr/bin/env python"

### CODE REVIEW

File: Library/Formula/svtplay-dl.rb [hunk 1]
```diff
@@ -22,6 +15,7 @@ class SvtplayDl < Formula
   end
 
   depends_on "rtmpdump"
+  depends_on "python"
```
Comment (DomT4) [inline Library/Formula/svtplay-dl.rb]: We don't set mandatory Homebrew Python dependencies, I'm afraid. It'll expect to use our Python, which means everyone has to install our Python, which isn't particularly fun or kind on a system that already keeps a pretty up-to-date Python around.

I guess you can add `depends_on :python => :optional` if @tdsmith isn't opposed, but I'm not sure on the upside here.

File: Library/Formula/svtplay-dl.rb [hunk 2]
```diff
@@ -1,15 +1,8 @@
...
   homepage "https://svtplay-dl.se"
-  url "https://pypi.python.org/packages/source/s/svtplay-dl/svtplay-dl-0.20.2015.11.29.tar.gz"
-  sha256 "7f93734da5c80387f354ccfa6cdd73af7a54978ac65ca65b1a29c801ac6dd685"
-
-  bottle do
-    cellar :any_skip_relocation
-    sha256 "3b26888e8aedd32827e2caa6ffcdeb2c115261662409dc0daf7ef2d6b4a842dc" => :el_capitan
-    sha256 "dc6e349be3fc0dd8ff7a396b0e7e2c480613af46a8c1643cff5a0bd5bfea59c8" => :yosemite
-    sha256 "1be2a31d2ce11561b486129ba2a2d6cf087d7126f02a91bb350c4d7a790c57b6" => :mavericks
-  end
```
Comment (bfontaine) [inline Library/Formula/svtplay-dl.rb]: Please don’t remove the `bottle do` block; our bot will update it. Thanks!

File: Library/Formula/svtplay-dl.rb (hunk 1 above)
Comment (spaam) [inline Library/Formula/svtplay-dl.rb]: python in some older version of OS X  dont have support for SNI. those users using that old version get an error message while accessing the sites even that python in brew have support for it. 
after some digging we found out that brew change my shebang from "#!/usr/bin/env python" to "#!/usr/bin/python". 

How can i prevent that to happen?

File: Library/Formula/svtplay-dl.rb (hunk 2 above)
Comment (spaam) [inline Library/Formula/svtplay-dl.rb]: okey :)

File: Library/Formula/svtplay-dl.rb (hunk 1 above)
Comment (DomT4) [inline Library/Formula/svtplay-dl.rb]: Which versions of OS X are we talking about?

Comment (spaam) [inline Library/Formula/svtplay-dl.rb]: 10.9 and 10.10.4 ..  looking at http://www.opensource.apple.com  i can see that in 10.10.5 have python 2.7.10

Comment (tdsmith) [inline Library/Formula/svtplay-dl.rb]: I think a better solution is to install the requests[security] dependencies, which will use PyOpenSSL with pyca/cryptography in preference to the built-in ssl module, which is not as flexible. Let me know if you need help enumerating those dependencies. Requests will automatically use them if they are available.

**Comment by spaam** [comment]
updated the repo with the review comments

**Comment by bfontaine** [comment]
There’s an issue in the `install` method and these resources will never be used. Right now the only one that’s used is `requests`.

It’d be great if you could fix that. You have to change the following block:
```rb
     %w[requests].each do |r|
       resource(r).stage do
         system "python", *Language::Python.setup_install_args(libexec/"vendor")
       end
     end
```

Into this one:
```rb    
    resources.each do |r|
       r.stage do
         system "python", *Language::Python.setup_install_args(libexec/"vendor")
       end
     end
```

Thanks! :heart:

**Comment by spaam** [comment]
thanks for the review. i will update the branch <3

File: Library/Formula/svtplay-dl.rb [hunk 3]
```diff
@@ -11,11 +11,59 @@ class SvtplayDl < Formula
...
+    url "https://pypi.python.org/packages/source/p/pyasn1/pyasn1-0.1.9.tar.gz"
+    sha256 "853cacd96d1f701ddd67aa03ecc05f51890135b7262e922710112f12a2ed2a7f"
+  end
+
+  resource "pycparser" do
+    url "https://pypi.python.org/packages/source/p/pycparser/pycparser-2.14.tar.gz"
+    sha256 "7959b4a74abdc27b312fed1c21e6caf9309ce0b29ea86b591fd2e99ecdf27f73"
+  end
+
+  resource "requests" do
```
Comment (DomT4) [inline Library/Formula/svtplay-dl.rb]: There are two `requests` resources here, one here and another below that's version `2.8.1`. Only need one :wink:

Comment (spaam) [inline Library/Formula/svtplay-dl.rb]: this is embarrassing..  will fix it :) thanks for the review :)

File: Library/Formula/svtplay-dl.rb [hunk 4]
```diff
@@ -25,11 +68,11 @@ class SvtplayDl < Formula
...
-    %w[requests].each do |r|
-      resource(r).stage do
-        system "python", *Language::Python.setup_install_args(libexec/"vendor")
-      end
-    end
+    resources.each do |r|
+       r.stage do
+         system "python", *Language::Python.setup_install_args(libexec/"vendor")
+       end
+     end
```
Comment (tdsmith) [inline Library/Formula/svtplay-dl.rb]: You need to add something like `touch libexec/"vendor/lib/python2.7/site-packages/ndg/__init__.py"` here for reasons with a long explanation explain (briefly, ndg is a namespace package and those are broken if they're installed outside of magical directories without this intervention)

Comment (spaam) [inline Library/Formula/svtplay-dl.rb]: okey, something more? :)

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/63c1c85678a0aed39dda44e5bf744f1d284413be. Thank you for another contribution to Homebrew @spaam! :smiley_cat:

**Comment by spaam** [comment]
thank you for the reviews! :)

**Pull request closed without merging**